src/cases/containers/case-viewer-container/case-viewer-container.component.spec.ts

```diff
@@ -32,7 +104,12 @@ describe('CaseViewerContainerComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  it('should return the two tabs', (done: DoneFn) => {
```

Comment:
Is there a test on the _actual_ `ccd-case-viewer` component to make sure it renders the tabs it receives? This doesn't test that anything within the HTML is wired up even with the mock `ccd-case-viewer` and I think it could benefit from at least testing that...

Reply:
`ccd-case-viewer` component is part of toolkit. Toolkit already tests if the component renders the tabs.

Reply:
Okay, cool. This still doesn't test that you've wired up the HTML, though. 😁

Reply:
Ah-ha! You've added another test beneath without even telling me. A nice surprise as I was just looking for the things you've _said_ you've addressed! 😁